Packs/ThreatGrid/Integrations/ThreatGrid/ThreatGrid.yml

```diff
@@ -552,7 +544,6 @@ script:
       description: File containing result
     description: Gets a specific threat feed
   - name: threat-grid-detonate-file
-    deprecated: true
```

Comment:
this one seems to have already been deprecated, please leave it as is (undo the deletion)

Reply:
?